### ISSUE TITLE
Is there a way to import existing ontologies?

### ISSUE THREAD

**Issue by MPParsley** [zazuko/ontology-manager#73]

**Comment by ktk** [comment]
Yes you can bootstrap with something existing, when you make sure that the N-Triples structure looks like what the Ontolog Manager creates itself.

The easiest way would probably be to get it working from scratch, start to add some sample classes and then make sure that the input you want to take as a base is put into the same form.

This is definitely not a documented process and it's too long ago that I remember details about the exact form it would have to have. But it should be possible, we did that in the past.

Please note that we have more a "schema.org" ontology than a Protégé ontology. So I would probably rather call it a "schema" than an ontology, as we use little to no OWL, just RDFS. And we use the schema.org domainIncludes/rangeIncludes by default as well.

**Comment by ktk** [comment]
@MPParsley bit OT but you might be interested in that post as well https://github.com/zazuko/ontology-manager/discussions/75
